src/main/java/svc/models/MunicipalityJudge.java

```diff
@@ -0,0 +1,8 @@
+package svc.models;
+
+public class MunicipalityJudge 
```

Comment:
I just don't see the value of having a "MunicipalityJudge" object at all. It's just a string, really. Especially if a Court just has 1 of them. Also, if it is indeed on the court, MunicipalityJudge as a name doesn't even make sense.